OpenEdXMobile/AndroidManifest.xml

```diff
@@ -387,4 +385,11 @@
 
     </application>
 
+    <queries>
```

Comment:
Please rebase the branch with master as this code is already implemented in another ticket

Reply:
`discovery/LEARNER-8822` is the base branch so we cover this suggestion at the time of the merge of `discovery/LEARNER-8822` to master.